docs/providers/terraform/azure.mdx

```diff
@@ -0,0 +1,126 @@
+---
+description: 'Terraform - Microsoft Azure provider for Nitric'
+---
+
+# Terraform Azure Provider Overview
+
+_The officially supported Nitric Terraform Azure Provider._
+
+```yaml
+provider: nitric/azuretf@latest
+```
+
+<Note>
+  The Nitric Terraform Azure Provider is currently in preview, it's not
+  recommended for production deployments. We recommend using the [Pulumi Azure
+  Providers](/providers/pulumi/azure) for production deployments until the
+  Terraform provider is stable.
+</Note>
+
+## Prerequisites
+
+The **Terraform CLI** is required to deploy the resulting Terraform Stack that Nitric generates. You can find the installation instructions for Terraform in the [Terraform documentation](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+
+The provider is built with the [Cloud Development Kit for Terraform (CDKTF)](https://developer.hashicorp.com/terraform/cdktf). Since CDKTF relies on Node.js, you'll need to have Node.js installed, you can read about the full CDKTF prerequisites in the [CDKTF documentation](https://developer.hashicorp.com/terraform/tutorials/cdktf/cdktf-install#prerequisites).
+
+## Enabling Nitric Terraform Providers
+
+The Nitric Terraform providers are currently in preview, to enable them you'll need to enable `beta-providers` in your Nitric project. You can do this by adding the following to your project's `nitric.yaml` file:
+
+```yaml title:nitric.yaml
+preview:
+  - beta-providers
+```
+
+## Azure Credentials
+
+The Terraform CLI typically uses standard Azure credential settings to authenticate with Azure. If you've set credentials for the Azure CLI or an Azure SDK previously, these settings should work without modification.
+
+If you're setting up your credentials for the first time, simply run azure login command and finish the login via your browser.
+
+```bash
+az login
+```
+
+Verify the Azure CLI install -
+
+```bash
+az -v
+```
+
+<Note>
+  See [Azure
+  documentation](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli)
+  for full details on credentials and configuration.
+</Note>
+
+### Azure CLI Installation
+
+Installing the Azure CLI assists with credentials setup. You can install it using these summarized instructions, for more options see the [Microsoft docs](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+<OSTabs>
+
+<TabItem label="Windows">
+
+Download & install the [latest CLI release](https://aka.ms/installazurecliwindows).
```

Comment:
Just a thought: Should this point to the documentation rather than being the install link? Don't know if people will trust downloading straight from our site if they're expecting to be taken to the Microsoft install docs. If that's the case, the pulumi provider doc should also be updated.

`https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-windows`

Reply:
That's fair, I'll update the original provider docs too then